js/components/closed-caption/fontSizeTab.js

```diff
@@ -1,9 +1,56 @@
-var React = require('react');
+var React = require('react'),
+    ClassNames = require('classnames'),
+    SelectionContainer = require('./selectionContainer');
 
 var FontSizeTab = React.createClass({
-  render: function(){
-    return(
-      <h4>Font Size content here</h4>
+  getInitialState: function() {
+    return {
+      selectedFontSize: this.props.closedCaptionOptions.fontSize
+    };
+  },
+
+  changeFontSize: function(fontSize) {
+    if (this.props.closedCaptionOptions.enabled){
+      this.props.controller.onClosedCaptionFontSizeChange(fontSize);
+      this.setState({
+        selectedFontSize: fontSize
+      });
+    }
+  },
+
+  setClassname: function(item, elementType) {
+    return ClassNames({
+      'oo-font-size-letter': elementType == "letter",
+      'oo-font-size-label': elementType == "label",
+      'oo-font-size-selected': this.props.closedCaptionOptions.fontSize == item && this.props.closedCaptionOptions.enabled,
+      'oo-font-size-label-selected': this.props.closedCaptionOptions.fontSize == item && this.props.closedCaptionOptions.enabled && elementType == "label",
+      'oo-disabled': !this.props.closedCaptionOptions.enabled
+    });
+  },
+
+  render: function() {
+    var fontSizes = ["Small", "Medium", "Large", "Extra Large"];
```

Comment:
Same idea here, could be part of the object state rather than reallocated every render.